quarkchain/cluster/client.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import sys
+import grpc
+
+sys.path.append("quarkchain/cluster")
+
+import grpc_client_pb2_grpc
+import grpc_client_pb2
+
+
+class GrpcClient(object):
+    def __init__(self, channel):
+        self.channel = channel
+        self.stub = grpc_client_pb2_grpc.ClusterSlaveStub(channel=self.channel)
+
+    def set_root_chain_confirmed_block(self) -> bool:
+        if self.channel == None:
+            self.channel = grpc.insecure_channel(
+                "localhost:50051"
+            )  # set default value for channel.
+        else:
+
+            request = (
```

Comment:
if channel is none them request will be undefined

Reply:
Done.